src/server/journal/streamer.cc

```diff
@@ -210,6 +210,10 @@ void RestoreStreamer::Run() {
   PrimeTable* pt = &db_array_[0]->prime;
 
   do {
+    if (pause_) {
```

Comment:
looks like you implemented the pause on the source node, but it should be on target node